libcovebods/data_reader.py

```diff
@@ -40,12 +56,13 @@ def get_all_data(self):
 
             with open(self._filename, "rb") as fp:
                 for statement in ijson.items(fp, "item"):
-                    statementType = (
-                        statement.get("statementType")
-                        if isinstance(statement, dict)
-                        and isinstance(statement.get("statementType"), str)
-                        else "unknown"
-                    )
+                    statementType = get_statement_type(statement)
+                    # statementType = (
```

Comment:
Is there a reason to leave the old code in?

Reply:
Removed